doc/03-monitoring-basics.md

```diff
@@ -1817,8 +1817,14 @@ Sometimes the problem in question should not be announced when the notification
 (the object reaching the `HARD` state), but after a certain period. In Icinga 2
 you can use the `times` dictionary and set `begin = 15m` as key and value if you want to
 postpone the notification window for 15 minutes. Leave out the `end` key -- if not set,
-Icinga 2 will not check against any end time for this notification. Make sure to
-specify a relatively low notification `interval` to get notified soon enough again.
+Icinga 2 will not check against any end time for this notification.
+
+> **Note**
+>
+> Setting the `end` key to `0` or a `negative` value will lead `Icinga2` to discard
+> all notifications, and you will never receive any notifications.
```

Comment:
That does not match the current code: https://github.com/Icinga/icinga2/blob/a018151eaf21fbec7c3f61208f96c96d0e93bf31/lib/icinga/notification.cpp#L299

A negative value should behave as if no value was set. I'd probably rephrase it something like "Setting end to 0 will disable the notification immediately when a problem occurs, effectively disabling the notification."